pkg/status/status_otlp.go

```diff
@@ -8,22 +8,25 @@
 package status
 
 import (
-	"github.com/DataDog/datadog-agent/cmd/agent/common"
+	"github.com/DataDog/datadog-agent/comp/otelcol/collector"
 	"github.com/DataDog/datadog-agent/pkg/config"
 	"github.com/DataDog/datadog-agent/pkg/otlp"
 )
 
+// OTLPCollector holds an instance of any running collector.
+// TODO: remove once this package is migrated to components.
+var OTLPCollector collector.Component
```

Comment:
Where is this variable being initialized ?

Reply:
`subommands/run.go`

Reply:
Added a comment.